includes/computer.h

```diff
@@ -62,14 +62,13 @@ struct _DisplayInfo {
     gchar *ogl_vendor, *ogl_renderer, *ogl_version;
     gboolean dri;
 
-    gchar *display_name, *vendor, *version;
-    gchar *extensions;
-    gchar *monitors;
-
     gint width, height;
 
     /* new stuff */
     xrr_info *xrr;
+
+    /* don't free */
+    gchar *vendor; /* points to xrr->xi->vendor */
```

Comment:
If somethings isn't supposed to be freed, it's best to just mark them as `const`.

Reply:
Ok.

Reply:
Note: making it const introduced a warning when sending it through info_*()